packages/messenger-widget/README.md

```diff
@@ -46,6 +46,7 @@ Follow the below given steps :-
     REACT_APP_RESOLVER_ADDR=0xae6646c22D8eE6479eE0a39Bf63B9bD9e57bAD9d
     REACT_APP_GENOME_REGISTRY_ADDRESS=0x5dC881dDA4e4a8d312be3544AD13118D1a04Cb17
     REACT_APP_PUBLIC_VAPID_KEY=BFCJLre0GeM6S-n4mkMX4SLZWlDR9qc8RsHyctsOPh_QDQkBuvCrMe9Rmq24736F-CJFp-3DkDWhp19X7mOJrEc
+    REACT_APP_NONCE=a3s8ej3js72jaj391
```

Comment:
Please use placeholder for the vars instead of actual values. That goes for all env vars that store some kind of key